test/unittest/xmlparser/XMLEndpointParserTests.cpp

```diff
@@ -59,11 +59,7 @@ class XMLEndpointParserTests : public ::testing::Test
     void helper_block_for_at_least_entries(
             uint32_t amount)
     {
-        std::unique_lock<std::mutex> lck(*xml_mutex_);
```

Comment:
I think this mutex is only used here. Remove it.

Reply:
done